Platform/QemuBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -196,9 +206,15 @@ BoardInit (
 )
 {
   EFI_STATUS            Status;
+  SBL_DEVICE_TABLE      *SblDeviceTable;
 
   switch (InitPhase) {
   case PreConfigInit:
+    SblDeviceTable = (SBL_DEVICE_TABLE *)AllocatePool (sizeof (SBL_DEVICE_TABLE) + sizeof (mSblDevices));
```

Comment:
can we use the same mechnism as other buffer, such as CfgData,  FlashMap, etc ?  In this way, it is consistent and easier to optimize later on.

Reply:
CfgData and FlashMap space will be available in Stage1A phase, it means it will use CAR memory if we uses same way. Just want to avoid to use CAR memory unless it is necessary since CAR memory is rare in size.